app/src/main/java/com/bigyoshi/qrhunt/qr/QrLibrary.java

```diff
@@ -14,14 +18,10 @@
  */
 public class QrLibrary implements Serializable {
 
-    private HashMap<String, PlayableQrCode> qrCodes;
     private String playerId;
     private transient FirebaseFirestore db;
-    private double lat;
-    private double lon;
-    private String qrHash;
-    private int score;
-    private PlayableQrCode qrCode;
+    private transient ArrayList<PlayableQrCode> qrCodesList;
+    private int scoreSorted; // 0 -> high-low, 1 -> low-high
```

Comment:
-1 and 1 for desc and asc are more canonical names for this, fine for now

Reply:
changed